selftests/unit/test_safeloader.py

```diff
@@ -80,6 +82,7 @@ def test_newlines(self):
     def test_enabled(self):
         self.assertTrue(safeloader.is_docstring_directive_enable(":avocado: enable"))
         self.assertTrue(safeloader.is_docstring_directive_enable(":avocado:\tenable"))
+        self.assertTrue(safeloader.is_docstring_directive_enable(":avocado: enable\n:avocado: tags=fast"))
```

Comment:
Suggestion for test (and possibly documentation):  what happens if this is given:

```
   :avocado: enable
   :avocado: disable
   :avocado: tags=fast
```